helpers/createAngleSideChainMultiBridge.sh

```diff
@@ -0,0 +1,200 @@
+#! /bin/bash
+
+source lib/utils/helpers/common.sh
+
+function chain_to_name {
```

Comment:
why don't we import it from utils?

Reply:
because I don't believe it will be used elsewhere